apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/BindingTests.cs

```diff
@@ -116,25 +105,46 @@ private async Task TestBindNonNull<T>(SpannerDbType parameterType, T value, Func
             {
                 Assert.Equal(value, valueRead);
             }
+
+            Assert.False(await reader.ReadAsync());
         }
 
+        private async Task<SpannerDataReader> GetBindingTestReader(SpannerConnection connection, SpannerDbType parameterType, object value)
+        {
+            using var cmd = connection.CreateSelectCommand("SELECT @v");
```

Comment:
I'm not sure we can dispose of the command either before reading through the data reader. If I remember correctly, not all data is returned inmediately, insteadt it's lazily fetched.

Reply:
Indeed. I suspect that disposing the command doesn't do much (possibly releases the session back to the pool?) but this is at least *logically* incorrect.

Given how short this is, I'd probably just inline it where you need it - but another option would be to pass in a delegate to do the reading/asserting while the command is open. Or just don't dispose the command, because I *believe* that SpannerDataReader.Dispose will dispose the command anyway.